packages/manager/modules/manager-components/src/resource-selector/index.js

```diff
@@ -0,0 +1,13 @@
+import angular from 'angular';
+import '@ovh-ux/ui-kit';
+
+import component from './component';
+import './index.scss';
+
+const moduleName = 'ovhManagerManagerComponentsResourceSelector';
+
+angular
+  .module(moduleName, ['oui'])
+  .component('ovhManagerResourceSelector', component);
```

Comment:
can we avoid ovh? just rename it to managerResourceSelector. This will avoid impact incase company name is changed.

Reply:
Done